integration/go/examples-go-new-tasks/tasks/a.go

```diff
@@ -0,0 +1,26 @@
+package tasks
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/zenaton/zenaton-go/v1/zenaton/task"
+)
+
+var A = task.NewCustom("TaskA", &AData{})
+
+type AData struct {
+	Workflow string
+}
+
+func (a *AData) Handle() (interface{}, error) {
+	fmt.Println("\n" + a.Workflow + ": Task A starts")
+	time.Sleep(3 * time.Second)
+	fmt.Println("\n" + a.Workflow + ": Task A ends")
+
+	return 0, nil
+}
+
+func (a *AData) Init(workflow string) {
+	a.Workflow = workflow
+}
```

Comment:
Yes actually, because these tasks are written differently. The tasks in the examples-go don't have the ability to accept imput. These do. They need to be able to accept imput to be able to write the name of the workflow as a prefix in their logs. That way I can filter by that prefix and find the output of just one example.